### PR TITLE
added postRequireTransforms

### PR DESCRIPTION
**Pull request by vinitm**
As per #116

### CODE REVIEW

**Comment by callumacrae** [comment]
Neat :)

File: index.js [hunk 1]
```diff
@@ -121,23 +123,35 @@ module.exports = function(options) {
     return requireName;
   }
 
+  function applyTransform(requireName, plugin) {
+    // var requireName = getRequireName(name);
+    var transform = postRequireTransforms[requireName];
+    // var plugin = requireFn(name);
```
Comment (jackfranklin) [inline index.js]: please could you remove the commented lines?

**Comment by jackfranklin** [comment]
I'm really sorry I completely missed this in my GitHub notifications! My apologies.

This looks really good; do you fancy adding a test?

**Comment by vinitm** [comment]
ya sure

**Comment by vinitm** [comment]
@jackfranklin is the test ok?

**Comment by callumacrae** [comment]
The test looks good to me :)

**Comment by jackfranklin** [comment]
Perfect 👍

**Comment by jackfranklin** [comment]
This has been published in 1.3.0

**Comment by vinitm** [comment]
great!!!

**Pull request merged**